Jenkinsfile

```
@@ -12,9 +12,10 @@ pipeline {
   }
   environment {
     DIRECTORY = "src/github.com/infobloxopen/ingress-nginx"
+    NGINX_VERSION = "v1.19.6"
```

This value duplicates that in the build.sh script, which opens up the possibility of the two diverging in the future. Something like this would keep the two in sync:
```suggestion
    NGINX_VERSION = sh(script: "cd ${DIRECTORY} && grep NGINX_VERSION= images/nginx/rootfs/build.sh | cut -d\= -f2",
                       returnStdout: true).trim()
```

---

good point! 👍